src/app/replayer/replayer.ml

```diff
@@ -1184,7 +1184,7 @@ module Snapp_helpers = struct
       let sgn =
         if Int64.is_negative body_data.balance_change then Sgn.Neg else Sgn.Pos
       in
-      Currency.Signed_poly.{ magnitude; sgn }
+      ({ magnitude; sgn } : _ Currency.Signed_poly.t)
```

Comment:
Because `[@@deriving fields]` on `Currency.Signed_poly.t` adds getter functions `magnitude` and `sgn`, which shadow the _values_ `magnitude` and `sgn` if you use the old version